packages/riipen-ui/src/components/Chip.jsx

```diff
@@ -30,21 +40,41 @@ class Chip extends React.Component {
      */
     disabled: PropTypes.bool,
 
+    /**
+     * Whether to invert colors on hover
+     */
+    hover: PropTypes.bool,
+
+    /**
+     * Icon to display at start of chip
+     */
+    iconStart: PropTypes.elementType,
+
     /**
      * The content of the label.
      */
     label: PropTypes.node,
 
     /**
-     * Callback function fired when the delete icon is clicked.
-     * If set, the delete icon will be shown.
+     * Action to perform when clicked
+     */
+    onClick: PropTypes.func,
+
+    /**
+     * Callback function fired when the icon is clicked.
      */
-    onDelete: PropTypes.func,
+    onIconClick: PropTypes.func,
```

Comment:
Since this is just named `onIconClick` it makes sense to also rename the `iconStart` to just `icon`.